.github/workflows/e2e-test.yaml

```diff
@@ -5,11 +5,9 @@ on:
     inputs:
       environment:
         description: "Select Dev or Test environment"
-        type: choice
+        required: true
+        type: string
```

Comment:
choice is a valid type, it just might need the required property
https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs

Reply:
it is for on dispatch but not for on call

https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype

I changed both for continuity sake